dev/com.ibm.ws.classloading/resources/com/ibm/ws/classloading/internal/resources/ClassLoadingServiceMessages.nlsprops

```diff
@@ -139,3 +139,19 @@ slf.no.acf.useraction=See the problem determination information on the WebSphere
 slf.failed.delete=CWWKL0072W: The [{0}] cache file cannot be deleted. This result can lead to the corruption of nested archives.
 slf.failed.delete.explanation=The system failed to delete the cache file. Because the file still exists in the system, this might cause corruption in nested archives.
 slf.failed.delete.useraction=Verify that the cache file is not in a locked state, which can cause the file deletion to fail.
+
+cls.classloader.config.duplicate=CWWKL0080E: API id [{0}] was already used in this list [{1}], remove the duplicate API id.
+cls.classloader.config.duplicate.explanation=Remove the duplicate API id.
+cls.classloader.config.duplicate.useraction=Check the server configuration for duplicate API id to ensure that the API id are correct.
+
+cls.classloader.config.not.allowed=CWWKL0081E: API id [{0}] is already added to the list when using prefix + or -, remove [{0}] from the list [{1}].
```

Comment:
They were different until I stopped allowing + or - with an API type that does not have a prefix.   For now just in case something changes, I would like to keep the message in the message file